ndex2/client.py

```diff
@@ -318,9 +318,40 @@ def post_multipart(self, route, fields, query_string=None):
 
     def save_new_network(self, cx, visibility=None):
         """
-        Create a new network (cx) on the server
 
-        :param cx: Network cx
+        Create a new network (cx) on the server
+       
+	 .. code-block:: python
+		
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#this method  creates a new network from cx, a python dict in CX format
+
+		# Query the network from my NDEx account, get small subset in cx format
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','INSERT NODE NAME HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client